julia/core.py

```diff
@@ -262,14 +262,16 @@ def juliainfo(runtime='julia'):
         [runtime, "-e",
          """
          println(VERSION < v"0.7.0-DEV.3073" ? JULIA_HOME : Base.Sys.BINDIR)
+         if VERSION >= v"0.7.0-DEV.3630"
+             using Libdl
+             using Pkg
+         end
          println(Libdl.dlpath(string("lib", splitext(Base.julia_exename())[1])))
          println(unsafe_string(Base.JLOptions().image_file))
          PyCall_depsfile = Pkg.dir("PyCall","deps","deps.jl")
          if isfile(PyCall_depsfile)
-            eval(Module(:__anon__),
-                Expr(:toplevel,
-                 :(Main.Base.include($PyCall_depsfile)),
-                 :(println(pyprogramname))))
+             include(PyCall_depsfile)
+             println(pyprogramname)
```

Comment:
I thought the reason for putting this in an anonymous module was to avoid putting `pyprogramname` into the global namespace.  Is this no longer a concern?

Reply:
I thought it was Julia 0.4 compatibility or something. This is executed in a subprocess which exits right after printing `pyprogramname`. So I don't think we have namespace problem. It is added in https://github.com/JuliaPy/pyjulia/commit/fde40d702a42a4aa5dda5dac873cbefbcc97baf3#diff-2f4f81856f2b92c6a4bb1e5bba2cf28aR260